microsoft-edge/extensions-chromium/developer-guide/native-client.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Native Client (NaCl)
+description: Native Client (NaCl).
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: extensions
+ms.date: 03/17/2021
+---
+<!-- Copyright A. W. Fuchs
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.  -->
+# Native Client (NaCl)
+
+Native Client (NaCl) was a sandbox for running compiled C and C++ code in the browser efficiently and securely, independent of the user's operating system.
+
+To enhance web security and performance, NaCl is deprecated for Microsoft Edge, along with other browsers.   
+
+<!-- todo: hard or soft deprection?  ie, will it still work, but discouraged?  or won't work now?  dates? -->
+
+The vibrant ecosystem around WebAssembly (Wasm) makes it a better fit for new and existing high-performance web apps. 
```

Comment:
We should link to webassembly docs so interested users can learn what it is.
https://webassembly.org/
https://developer.mozilla.org/docs/WebAssembly

Reply:
```suggestion
To run compiled code in your extension, we recommend using WebAssembly instead. WebAssembly, also known as Wasm, is a low-level assembly-like language which many other languages, such as C, C++, or Rust, can compile to, and which can run at near native-speed in web browsers.

See also:

* [WebAssembly](https://developer.mozilla.org/docs/WebAssembly) at MDN
* [webassembly.org](https://webassembly.org/)
```